geode-for-redis/src/main/java/org/apache/geode/redis/internal/commands/RedisCommandType.java

```diff
@@ -286,6 +289,12 @@
   ZUNIONSTORE(new ZUnionStoreExecutor(), SUPPORTED,
       new Parameter().min(4).flags(WRITE, DENYOOM, MOVABLEKEYS)),
 
+  /************** Lists *****************/
+
+  LLEN(new LLenExecutor(), SUPPORTED, new Parameter().exact(2).flags(READONLY, FAST)),
+  LPOP(new LPopExecutor(), SUPPORTED, new Parameter().min(2).flags(WRITE, FAST)),
```

Comment:
Currently, I have the LPOP executor check for more than two args and throw the appropriate error there. (Marked with a TODO comment, but it'll be found pretty fast as soon as we implement Redis 6.2+ semantics.) I think it better to keep the 'special case' logic where LPOP is implemented and tested.

Reply:
This will result in incorrect information being returned by the COMMAND command though, since that parses the contents of `RedisCommandType` to generate its output. We only accept exactly 2 arguments for the LPOP command, so that's what should be returned by the COMMAND command, and hence what should be set here.